src/components/Petitions/index.js

```diff
@@ -1,38 +1,43 @@
 import React from 'react';
 import settings from 'settings';
+import { Link } from 'react-router';
 import Container from 'components/Container';
 import Grid from 'components/Grid';
 import GridItem from 'components/GridItem';
+import TextCenter from 'components/TextCenter';
 import Teaser from 'components/Teaser';
 import Header from 'components/Header';
 import PageTitle from 'components/PageTitle';
-import ButtonLink from 'components/ButtonLink';
+import Autocomplete from 'containers/Autocomplete';
 import Loading from 'components/Loading';
 
-const Petitions = ({ petitions }) => (
+const Petitions = ({ petitions, autocompleteProps }) => (
   <Container>
     <section>
       <Header>
         <PageTitle
           title={settings.petitionsPage.title}
           centered>
-          <ButtonLink
-            href={'/petitions/new'}
-            text={settings.petitionsPage.createButton.text}
-            modifier={'accent'}
-          />
+
+          <Autocomplete {...autocompleteProps} />
```

Comment:
Will probably be moved outside of the page title for more styling or to put it in a PetitionFilters component.